libafl/src/stages/calibrate.rs

```diff
@@ -75,7 +75,6 @@ pub struct CalibrationStage<C, E, O, OT> {
     stage_max: usize,
     /// If we should track stability
     track_stability: bool,
-    restart_helper: ExecutionCountRestartHelper,
```

Comment:
How can executioncount ever get stuck?

Reply:
i don't know.
it's just stuck forever for me for and with this patch it's fixed

Reply:
Then we need to fix retrying